src/test/java/com/imsweb/validation/functions/MetafileContextFunctionsTest.java

```diff
@@ -1166,6 +1166,217 @@ public void testGEN_ILOOKUP() throws ConstructionException {
     }
 
     @Test
+    public void testGEN_SQLLOOKUP() {
+
+        // define the table we are going to use
+        List<List<Object>> table = new ArrayList<>();
+        List<Object> row0 = new ArrayList<>();
+        row0.add("GPCODE");
+        row0.add("GPNAME");
+        row0.add("SITELOW");
+        row0.add("SITEHIGH");
+        table.add(row0);
+        List<Object> row1 = new ArrayList<>();
+        row1.add("15");
+        row1.add("Bones");
+        row1.add("C420");
+        row1.add("C429");
+        table.add(row1);
+        List<Object> row2 = new ArrayList<>();
+        row2.add("17");
+        row2.add("Skin");
+        row2.add("C440");
+        row2.add("C449");
+        table.add(row2);
+        List<Object> row3 = new ArrayList<>();
+        row3.add("18");
+        row3.add("Another Bones");
+        row3.add("C400");
+        row3.add("C409");
+        table.add(row3);
+
+        List<Object> columns = new ArrayList<>();
+        Assert.assertTrue(_functions.GEN_SQLLOOKUP(table, columns, ""));
```

Comment:
We are good. I thought the empty value caused it to return true, didn't realize the columns were empty as well. This is resolved.